projects/demo/src/modules/markup/lists/examples/1/index.html

```diff
@@ -1,27 +1,50 @@
-<ul class="tui-list tui-list_large">
-    <li class="tui-list__item">Code Review ($ 10)</li>
-    <li class="tui-list__item">
-        The whole project refactoring ($ 1000)
-        <ul class="tui-list tui-list_linear tui-list_nested">
-            <li class="tui-list__item">Add auto prettier and linters on precommit ($ 50)</li>
-            <li class="tui-list__item">Rewrite to react and back ($ Infinity)</li>
+@for (platform of platforms; track $index) {
+    <div
+        class="wrapper"
+        [tuiPlatform]="platform"
+    >
+        <code>{{ platform === 'web' ? 'Web' : 'Android/iOS' }}</code>
+
+        <h2>Body L</h2>
+
+        <ul
+            aria-label="List of Continents"
```

Comment:
Not sure we want to support it as official API of lists, let's not add this behavior